R/pkg/R/functions.R

```diff
@@ -1697,8 +1697,8 @@ setMethod("to_date",
           })
 
 #' @details
-#' \code{to_json}: Converts a column containing a \code{structType}, array of \code{structType},
-#' a \code{mapType} or array of \code{mapType} into a Column of JSON string.
+#' \code{to_json}: Converts a column containing a \code{structType}, a \code{mapType}
+#' or an array into a Column of JSON string.
```

Comment:
does `\code{arrayType}` work here?

Reply:
I am not sure

Reply:
it should
could we add some tests for this in R?

Reply:
Let's add one simple python doctest as well

Reply:
I added tests for Python and R. Please, take a look at them.